quest/solvers.py

```diff
@@ -1,3 +1,25 @@
 """
 This is a file where DIIS and CG would go.
 """
+
+def DIIS_step(state_list, error_list, diis_vectors = 6):
+    """Routine that returns a guess for a new state vector
+    using DIIS
```

Comment:
Lets up these doc strings before we merge it in. Check [here](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) for a guide.